packages-oss.yml

```diff
@@ -241,7 +241,7 @@ layers:
   - name: build-static-assets
     source-include: Makefile
     dockerfile: |-
-      RUN go-bindata-assetfs -o bindata_assetfs.go -pkg http -prefix pkg -modtime 1480000000 -tags ui ./pkg/web_ui/...
+      RUN go-bindata-assetfs -o bindata_assetfs.go -pkg http -prefix pkg -modtime 1480000000 -tags ui ./http/web_ui/...
```

Comment:
Do we still need to run go-bindata-assetfs with this new system?

Reply:
No. I believe this PR needs some additional polishing, it was time-boxed and functional when I opened it. I had planned on reviewing and continuing some cleanup depending on interest.

Reply:
These parts should be cleaned up now